.github/workflows/wheels.yml

```diff
@@ -80,6 +80,51 @@ jobs:
         name: cibw-wheels-${{ matrix.os }}
         path: wheelhouse/*.whl
 
+  build_wheels_windows:
+    name: Wheels on windows
+    runs-on: windows-latest
+    env: 
+        USERNAME: pcl-python-bindings
+        VCPKG_EXE: c:\\vcpkg\\vcpkg.exe
+        FEED_URL: https://nuget.pkg.github.com/pcl-python-bindings/index.json
+        VCPKG_BINARY_SOURCES: "clear;nuget,https://nuget.pkg.github.com/pcl-python-bindings/index.json,readwrite"
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: true
+
+    - name: Bootstrap vcpkg
+      env:
+        VCPKG_DEFAULT_TRIPLET: x64-windows
+      run: |
+        cd "$VCPKG_INSTALLATION_ROOT"
+        git pull > nul
+        ./bootstrap-vcpkg.bat -disableMetrics
+      shell: bash
+
+    - name: 'Setup NuGet Credentials'
+      shell: pwsh
+      run: >
+        .$(${{ env.VCPKG_EXE }} fetch nuget) `
+          sources add `
+          -Source "${{ env.FEED_URL }}" `
+          -StorePasswordInClearText `
+          -Name GitHubPackages `
+          -UserName "${{ env.USERNAME }}" `
+          -Password "${{ secrets.GH_PACKAGES_TOKEN }}"
+        .$(${{ env.VCPKG_EXE }} fetch nuget) `
+          setapikey "${{ secrets.GH_PACKAGES_TOKEN }}" `
+          -Source "${{ env.FEED_URL }}"
+
+    - uses: pypa/cibuildwheel@v2.23
+      env:
+        SKBUILD_CMAKE_ARGS: "-DCMAKE_TOOLCHAIN_FILE=c:/vcpkg/scripts/buildsystems/vcpkg.cmake"
+        CIBW_SKIP: "cp38-win_amd64" # Skip CPython 3.8 for now, as test fails.
```

Comment:
I think it is PyPy 3.8, not CPython 3.8, that fails? The same thing happened on Linux and MacOS, so I disabled the `pp*-` builds there.